src/test/java/step3/model/lotto/LottoTest.java

```diff
@@ -4,24 +4,37 @@
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
-class LottoTest {
+import org.junit.jupiter.params.provider.ValueSource;
 
+class LottoTest {
+    private List<LottoNumber> changeToLottoNumbers(List<Integer> numbers){
+        return numbers.stream().map(LottoNumber::new).collect(Collectors.toList());
+    }
     @Test
-    void 정상_생성_프린트_검증() {
-        Lotto lotto = new Lotto(Arrays.asList(1,2,3,4,5,6));
-        assertThat(lotto.toString()).isEqualTo("[1, 2, 3, 4, 5, 6]");
+    void 중복된_입력() {
+        assertThatThrownBy(() -> new Lotto(Stream.of(1,1,3,4,5,6).map(LottoNumber::new).collect(Collectors.toList())))
```

Comment:
```suggestion
        List<LottoNumber> lottoNumbers = Stream.of(1,1,3,4,5,6).map(LottoNumber::new).collect(Collectors.toList());
        assertThatThrownBy(() -> new Lotto(lottoNumbers))
```
하나의 예외 포인트만을 테스트하는게 좋습니다.